route.go

```diff
@@ -3,33 +3,130 @@ package netlink
 import (
 	"fmt"
 	"net"
+	"strings"
 	"syscall"
 )
 
-// Scope is an enum representing a route scope.
-type Scope uint8
-
-const (
-	SCOPE_UNIVERSE Scope = syscall.RT_SCOPE_UNIVERSE
-	SCOPE_SITE     Scope = syscall.RT_SCOPE_SITE
-	SCOPE_LINK     Scope = syscall.RT_SCOPE_LINK
-	SCOPE_HOST     Scope = syscall.RT_SCOPE_HOST
-	SCOPE_NOWHERE  Scope = syscall.RT_SCOPE_NOWHERE
-)
```

Comment:
Is there a reason you changed Scope from an enum? I like having the values here so the caller doesn't have to use the syscall package to pass values in. Tables and Protocols can be done the same way and the string methods can be directly on the enum. See how it is done in xfrm.go: https://github.com/vishvananda/netlink/blob/master/xfrm.go#L39